utils/file.js

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs-extra';
+import { resolve } from 'path';
+
+/**
+ * JS file containing all helpers for working with files
+ */
+
+/**
+ * Returns all files within a directory, regardless of directory structure.
+ *
+ * @param dir - root directory
+ *
+ * @returns {Promise<string[]>}
+ */
+export async function getFiles(dir) {
+  let dirents = await fs.readdir(dir, {withFileTypes: true});
+  /**
+   * NOTE: filter out hidden files (like: .gitkeep)
+   *       {@link https://stackoverflow.com/questions/8905680/nodejs-check-for-hidden-files/20285137#20285137}
+   */
+  dirents = dirents.filter(dir => !(/(^|\/)\.[^\/\.]/g).test(dir.name));
+  const files = await Promise.all(dirents.map((dirent) => {
+    const res = resolve(dir, dirent.name);
+    return dirent.isDirectory() ? getFiles(res) : res;
+  }));
+  return Array.prototype.concat(...files);
+}
+
+export function getDirectoriesSync(source) {
```

Comment:
Do we need these? (I would expect now the paths to be hard coded, and after that you get them from the DB)

Reply:
That whole file should be removed, shouldnt have pushed it since it isnt relevant at the moment. semi-hardcoded url is here: 
https://github.com/lblod/lpdc-management-service/blob/508316f9c50ab13088650b9955f1c175b7aa66e7/lib/getForm.js#L19